script/test

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Should we name the file `test.sh`? (i.e. with a .sh suffix) I'm not sure what is best practice here really.

Reply:
We had `script/start` already so followed that pattern, which I think is inspired by https://github.blog/2015-06-30-scripts-to-rule-them-all/.